usr/local/bin/install-ukca-data

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Install UKCA input data:
+# 1) wget tar file from JASMIN
+# 2) Extract input files from tar file
+# 3) Run md5sum on files to check for data corruption
+# 4) Copy to umdir
+# 5) Clean-up
+
+if [ "$#" -gt 1 ]; then
+  echo "Usage: $(basename $0) [version]"
+  exit 1
+fi
+
+ver=CUR
+if [[ -n "$1" ]]; then
+  ver=$1
+fi
+
+tmpdir=$(mktemp -d)
+cd $tmpdir
+
+tarfile=ukca_ancils
+
+echo "Extracting input data archive from JASMIN UKCA GWS"
+wget -q --show-progress http://gws-access.ceda.ac.uk/public/ukca/${tarfile}_v${ver}.tgz
+if [ $? -ne 0 ]; then
+  echo "Error downloading UKCA data"
+  exit 1
+fi
+
+tar -xzf ${tarfile}_v${ver}.tgz
+
+# now md5 sum the files to make sure there has been no corruption
+echo "Checking files..."
+md5sum --status -c ukca_ancils.md5
+if [ $? -ne 0 ]; then
+  echo "Error extracting UKCA files"
+  exit 1
+fi
+
+# If a central install (using rose-stem) has been attempted before the ancils
+# have been set up, this could have left a dead symlink which needs removing:
+if [[ -h "$HOME/umdir/ancil" ]]; then
+  rm -f $HOME/umdir/ancil
+fi
+
+mkdir -p $HOME/umdir
+cp -r $tarfile/* $HOME/umdir
+
+# Clean up
+rm -rf $tmpdir
+
+echo
+echo "UKCA Inputs have been installed to $HOME/umdir/ancil"
+echo "and $HOME/umdir/standard_jobs"
```

Comment:
Yes I meant $HOME/umdir/standard_jobs - I thought that I made my comment  against line 56 but it doesn't seem to display like that.

Actually I'd mis-read what the script is doing so I'll merge this as is. At least we both get some practise with pull requests!